backend/src/systemInfo.ts

```diff
@@ -0,0 +1,18 @@
+import environment from './environment'
+import cli from './cliInterface'
+import lan from './LAN'
+import os from 'os'
+
+export default async function systemInfo() {
+  await lan.getInterfaces()
+
+  let info: ILookup = {
```

Comment:
Probably just return here instead of allocating a variable?

Reply:
Yep! missed that one thanks!